testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveViews.java

```diff
@@ -172,4 +172,88 @@ public void testLateralViewJsonTupleAs()
                 "SELECT * FROM test_json_tuple_view",
                 queryAssert -> queryAssert.containsOnly(row(3, "Mateusz", "Gajewski", "true", "1000", null, null)));
     }
+
+    @Test(groups = HIVE_VIEWS)
+    public void testFromUtcTimestamp()
+    {
+        onTrino().executeQuery("DROP TABLE IF EXISTS test_from_utc_timestamp_source");
+        onHive().executeQuery("CREATE TABLE test_from_utc_timestamp_source AS SELECT " +
+                "  CAST(123 AS tinyint) source_tinyint, " +
+                "  CAST(10123 AS smallint) source_smallint, " +
+                "  CAST(2592000123 AS integer) source_integer, " +
+                "  CAST(2592000123 AS bigint) source_bigint, " +
+                "  CAST(2592000.0 AS float) source_float, " +
+                "  CAST(2592000.123 AS double) source_double, " +
+                "  CAST(2592000.123 AS decimal(10,3)) source_decimal_three," +
+                "  CAST(2592000 AS DECIMAL(10,0)) source_decimal_zero," +
+                "  timestamp '1970-01-30 16:00:00' source_timestamp, " +
+                "  date '1970-01-30 16:00:00' source_date ");
```

Comment:
Hive allows this? Does it truncate parts smaller than a day?

Reply:
Oh - did not notice that. I meant `date '1970-01-30'`. But apparently Hive is not very picky about the literal :)

Reply:
But it seems we are handling it correctly. Please keep it and add another case with the "correct" literal.

Reply:
I hope it doesn't matter, _how_ a date value is created. I.e. i do hope `date '1970-01-30 16:00:00'` is not distinguishable from `date '1970-01-30'`